src/resultspage.cpp

```diff
@@ -212,6 +230,24 @@ void ResultsPage::onJumpToCallerCallee(const Data::Symbol& symbol)
     ui->resultsTabWidget->setCurrentWidget(m_resultsCallerCalleePage);
 }
 
+void ResultsPage::setData(const Data::Symbol& data)
```

Comment:
the name of this function is pretty bad, I would just remove it and inline the call below instead to where it is currently being called -  that it makes it clear what this is doing